File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/bass/BAS0101U00PrBasGridColumnChangedItemInfo.java
package nta.med.data.model.ihis.bass;

public class BAS0101U00PrBasGridColumnChangedItemInfo {
	private String dupYn;
	private String error;
	public BAS0101U00PrBasGridColumnChangedItemInfo(){}
	public BAS0101U00PrBasGridColumnChangedItemInfo(String dupYn,
			String error) {
		super();
		this.dupYn = dupYn;
		this.error = error;
	}
	public String getDupYn() {
		return dupYn;
	}
	public void setDupYn(String dupYn) {
		this.dupYn = dupYn;
	}
	public String getError() {
		return error;
	}
	public void setError(String error) {
		this.error = error;
	} 
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Bass/BAS0101U00PrBasGridColumnChangedItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Bass
{
    [Serializable]
    public class BAS0101U00PrBasGridColumnChangedItemInfo
    {
        private String _dupYn;
        private String _error;

        public String DupYn
        {
            get { return this._dupYn; }
            set { this._dupYn = value; }
        }

        public String Error
        {
            get { return this._error; }
            set { this._error = value; }
        }

        public BAS0101U00PrBasGridColumnChangedItemInfo() { }

        public BAS0101U00PrBasGridColumnChangedItemInfo(String dupYn, String error)
        {
            this._dupYn = dupYn;
            this._error = error;
        }

    }
}